pkg/scheduler/plugins/topology/job_filtering.go

```diff
@@ -48,8 +51,10 @@ func (t *topologyPlugin) subSetNodesFn(job *podgroup_info.PodGroupInfo, tasks []
 	}
 
 	if maxAllocatablePods < len(tasks) {
-		log.InfraLogger.V(6).Infof("no relevant domains found for job %s, workload topology name: %s",
-			job.PodGroup.Name, topologyTree.Name)
+		job.SetJobFitError(
+			podgroup_info.PodSchedulingErrors,
+			fmt.Sprintf("No relevant domains found for job %s, workload topology name: %s", job.Name, topologyTree.Name),
```

Comment:
It will be recorded on the workload, I don't the name of the workload is not really needed. It will be part of the metadata anyway.

Let's just say:
```
No relevant domains found for workload in topology tree: %s
```